aspnetcore/tutorials/min-web-api.md

```diff
@@ -898,29 +946,32 @@ A DTO may be used to:
 
 To demonstrate the DTO approach, update the `Todo` class to include a secret field:
 
-[!code-csharp[](min-web-api/samples/6.x/todoDTO/Program.cs?name=snippet_secret&highlight=6)]
+:::code language="csharp" source="min-web-api/samples/7.x/todoDTO/Program.cs" id="snippet_secret" highlight="6":::
 
 The secret field needs to be hidden from this app, but an administrative app could choose to expose it.
 
 Verify you can post and get the secret field.
 
 Create a DTO model:
 
-[!code-csharp[](min-web-api/samples/6.x/todoDTO/Program.cs?name=snippet_DTO)]
+:::code language="csharp" source="min-web-api/samples/7.x/todoDTO/Program.cs" id="snippet_DTO":::
 
 Update the code to use `TodoItemDTO`:
```

Comment:
The following code shows `#endif`
![image](https://user-images.githubusercontent.com/3605364/191873970-3598efba-f7f9-4b4b-bdae-58caf101a6fd.png)

Also, let's get those awful classes out of program.cs